targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT/GeneratedSource/cycfg_notices.h

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+* File Name: cycfg_notices.h
+*
+* Description:
+* Contains warnings and errors that occurred while generating code for the
+* design.
+* This file should not be modified. It was automatically generated by 
+* ModusToolbox 1.0.0
+* 
+********************************************************************************
+* Copyright (c) 2017-2018 Cypress Semiconductor.  All rights reserved.
+* You may use this file only in accordance with the license, terms, conditions, 
```

Comment:
What license for these files? It's missing in this header (it references user agreement, cant spot any). There were similar files already integrated, they were released under apache 2.0

Reply:
The low-level PSoC initialization code in Cypress port is generated by ModusToolbox MCU Configurators (unlike FUTURE_SEQUANA port which used PSoC Creator tool). The configurators are being updated to generate code with Apache license header.
I will regenerate the code with latest configurators to apply the new license. No Cypress EULA included in mbed port, this is intended.

Reply:
@0xc0170: this is addressed now, all BSP code in GeneratedSource is now Apache 2.0.

Reply:
@0xc0170: together with the latest rebase, I included the CapSense middleware: 
5252d05. The code is supplied with the standard Cypress EULA, it is not Apache 2.0 licensed. Per the guidelines, the license file is added to the psoc6mw/capsense folder.